.github/workflows/extendJSONFile.yml

```diff
@@ -15,16 +15,26 @@ on:
         default: 'https://raw.githubusercontent.com/DSACMS/repo-scaffolder/main/tier1/%7B%7Bcookiecutter.project_slug%7D%7D/repolinter.json'
         required: true
         type: string
+    outputs:
+      raw-json: 
+        description: 'JSON of the file'
+        value: ${{ jobs.extend-json.outputs.raw-json }}
 
 jobs:
   extend-json:
     runs-on: ubuntu-latest
     name: Extend JSON file with extends property
     outputs: 
-      raw-json: ${{ steps.extend-json.outputs.RAW_JSON }}
+      raw-json: ${{ steps.extend-json.outputs.raw-json }}
     steps:
+      - name: Checkout repository
```

Comment:
the repo needs to be checked out before it can run the script. It is checked out into the `workflows` dir